nixos/modules/services/networking/prosody.nix

```diff
@@ -336,10 +336,10 @@ let
         '';
       };
 
-      vcard_muc = mkOption {
-        type = types.bool;
-        default = true;
-      description = lib.mdDoc "Adds the ability to set vCard for Multi User Chat rooms";
+      extraModules = mkOption {
```

Comment:
Lists tend to be annoying because they make it very difficult to remove items from the default value. Can this be an attrset instead with e.g. `"muc_mam".enable = true;`?

(I actually don't know if we have something better as a common pattern for this in NixOS - I just know lists are annoying and that's how I would intuitively have go at fixing this.)